services/ui-src/src/components/reports/ReportProvider.tsx

```diff
@@ -121,33 +122,32 @@ export const ReportProvider = ({ children }: Props) => {
         selectedState
       );
 
-      if (workPlanSubmissions?.length !== sarSubmissions?.length) {
-        let lastFoundSubmission: ReportMetadataShape | undefined = undefined;
-        workPlanSubmissions.forEach((submission: ReportMetadataShape) => {
+      let lastFoundSubmission: ReportMetadataShape | undefined = undefined;
```

Comment:
This will be abstracted in SAR Form Creation Cleanup Part 2, so please don't mind that its without comments and living large for now :D